package.json

```diff
@@ -27,6 +28,7 @@
   },
   "dependencies": {
     "@types/date-fns": "^2.6.0",
-    "date-fns": "^2.29.3"
+    "date-fns": "^2.29.3",
+    "uuid": "^9.0.0"
```

Comment:
We don't need to do it now, but we should move all these to the dev-dependencies for consistency (everything in this repo is dev)